src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int countFigures = 5;
+    private static final int defaultRadius = 10;
+    private static final int bound = 50;
```

Comment:
All `private static final` fields are constants and should have name in UPPERCASE
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.